api/src/org/apache/cloudstack/api/response/SnapshotResponse.java

```diff
@@ -82,6 +82,10 @@
     @Param(description = "valid types are hourly, daily, weekly, monthy, template, and none.")
     private String intervalType;
 
+    @SerializedName(ApiConstants.LOCATION_TYPE)
+    @Param(description = "valid location types are primary and archive.")
+    private String locationType;
```

Comment:
LocationType is a String in response. It makes sense for it be String in request as well.